pandas/io/parsers.py

```diff
@@ -221,8 +221,9 @@ class ParserWarning(Warning):
 
 _sep_doc = """sep : str, default {default}
     Delimiter to use. If sep is None, will try to automatically determine
-    this. Regular expressions are accepted and will force use of the python
-    parsing engine and will ignore quotes in the data."""
+    this. Separators longer than 1 character will be interpreted as regular
+    expressions, will force use of the python parsing engine and will ignore
```

Comment:
maybe show an example, as ``sep='\s+'`` DOES use the c-engine (its a bit of a special case), an example would be ``sep=`\r\t``` or somesuch